presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -103,6 +103,21 @@ class TaskManager {
       std::optional<protocol::Duration> maxWait,
       std::shared_ptr<http::CallbackRequestHandlerState> state);
 
+  /// Update buffer count for a given task for broadCast case.
+  /// Returns true, if update was successful, else returns false.
+  /// Also, returns true for a no-op case.
+  /// Note: This method is made public for unit testing purpose only.
+  inline bool updateBroadcastOutputBuffers(
+      std::shared_ptr<PrestoTask>& prestoTask,
```

Comment:
It doesn't seem like you are using prestoTask in this function. Why are you passing it ?